.gitignore

```diff
@@ -6,6 +6,9 @@ data/*/
 !data/csvs/
 data/csvs/*/
 saved_models/
+*.ipynb_checkpoints/
+examples/*
```

Comment:
Why should we ignore files in the `examples` directory? Isn't a main point of this PR to add files to this directory?

Reply:
You're right, I want to ignore subdirectories of that folder.